tests/Event/CommandHandlers/CopyEventHandlerTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\CommandHandlers;
+
+use Broadway\Domain\DomainMessage;
+use CultuurNet\UDB3\Calendar;
+use CultuurNet\UDB3\CalendarType;
+use CultuurNet\UDB3\Event\Commands\CopyEvent;
+use CultuurNet\UDB3\Event\Event;
+use CultuurNet\UDB3\Event\EventRepository;
+use CultuurNet\UDB3\Event\Events\EventCopied;
+use CultuurNet\UDB3\Event\EventType;
+use CultuurNet\UDB3\Event\ValueObjects\LocationId;
+use CultuurNet\UDB3\Language;
+use CultuurNet\UDB3\Model\ValueObject\Calendar\OpeningHours\OpeningHours;
+use CultuurNet\UDB3\Model\ValueObject\Calendar\PermanentCalendar;
+use CultuurNet\UDB3\Timestamp;
+use CultuurNet\UDB3\Title;
+use DateTimeImmutable;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class CopyEventHandlerTest extends TestCase
```

Comment:
I guess it was not possible to use `CommandHandlerScenarioTestCase` here because two aggregates are involved when doing a copy?

Reply:
Yeah I tried but the problem is indeed that the Scenario expects to start from an Aggregate that exists, and then allows you to compare the new events on that aggregate to the ones you'd expect. And in this case the Aggregate that "exists" is the original event, but we want to check the events on the new Aggregate.

Reply:
Thanks for clarifying!